java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/RuntimeImageBuildStep.java

```diff
@@ -47,7 +47,7 @@ protected RuntimeImageBuildStep(JdkServerLookup jdkServerLookup, String compatIm
   @Override
   public void run(BuildContext buildContext) throws BuildStepException {
     Artifact artifact = getArtifact(buildContext);
-    logger.debug("Found Java artifact {}", artifact);
+    logger.debug("Identified Java artifact for deployment {}", artifact);
```

Comment:
`info` level maybe? It seems pretty important, especially if it's auto-discovered.